graphium/cli/train_finetune.py

```diff
@@ -44,6 +44,14 @@ def run_training_finetuning(cfg: DictConfig) -> None:
 
     st = timeit.default_timer()
 
+    # Disable wandb if the user is not logged in.
+    if wandb.login() is False:
```

Comment:
Nice! I like this! Could we merge it with the logic I introduced (i.e. checking if the `constants.wandb` key exists)? Or do both approaches complement each other?

Reply:
I think this is useful for certain environments where the user may not be logged in but the config is set, especially where we are both using configs for experiments + public release. But no strong feelings about exactly the mode to check, but this seemed simple enough.

Reply:
Yeah I think you're right! I'll make it a two part with an `elif`.